avocado/plugins/json_variants.py

```diff
@@ -18,11 +18,25 @@
 from avocado.core import exit_codes, varianter
 from avocado.core.future.settings import settings
 from avocado.core.output import LOG_UI
-from avocado.core.plugin_interfaces import CLI, Varianter
+from avocado.core.plugin_interfaces import CLI, Init, Varianter
 
 _NO_VARIANTS = -1
 
 
+class JsonVariantsInit(Init):
+
+    name = 'json_variants'
+    description = "JSON serialized based varianter initialization"
+
+    def initialize(self):
+        help_msg = 'Load the Variants from a JSON serialized file'
+        settings.register_option(section='json.variants',
+                                 key='load',
+                                 default=None,
+                                 help_msg=help_msg,
+                                 long_arg='--json-variants-load')
+
```

Comment:
IMO, if there is no parser here, don't need to pass `long_arg`. This will be done with `add_argparser_to_option()` right?

Reply:
You're right... that was a mistake.  I'll force push if there's nothing else to change.